content/programs/project-ideas-collection/modernize-cc-resource-archive/contents.lr

```diff
@@ -9,17 +9,17 @@ The Resources Archive has an outdated visual setup, and needs functionality
 fixes and improvements.
 ---
 expected_outcome:
-Upgrade the look & feel to match current CC aesthetics, using the internal
+Upgrade the look and feel to match current CC aesthetics, using the internal
 Design System (Vocabulary). Codebase is running current and standards compliant
-technology implementations. Entire site builds as static files on GH pages.
+technology implementations. Entire site builds as static files on GitHub Pages.
 Generate clear documentation of codebase and functionality.
 ---
 contributor_tasks:
 Contributor will refactor codebase to fix resource submission, use current
 semantic, accessible, and standards compliant HTML, CSS, and JavaScript to
 match the current visual aesthetic of Creative Commons sites. Improve the
 overall UX for finding, submitting, and  accessing resources. Ensure the site
-functions as a static and stable Jekyll site hosted on GH Pages as static
+functions as a static and stable Jekyll site hosted on GitHub Pages as static
```

Comment:
Since we are open to other generators, I missed that we should rephrase this.

> functions as a static and stable Jekyll site hosted on GitHub Pages as static

to this: 

> functions as a static and stable site hosted on GitHub Pages